pkgs/tools/admin/awsweeper/default.nix

```diff
@@ -1,17 +1,22 @@
-{ lib, buildGoModule, fetchFromGitHub }:
+{ lib, buildGoModule, fetchurl, fetchFromGitHub }:
 
 buildGoModule rec {
   pname = "awsweeper";
   version = "0.7.0";
 
+  patches = fetchurl {
+    url = "https://raw.githubusercontent.com/c00w/patches/master/awskeeper.patch";
```

Comment:
I also would like a comment here. It is hard for me to review 10MB patches.
Also it would be interesting why our vendoring approach does not work here.

Reply:
mockgen Doesn't like it - added comment.